i18n-helpers/src/normalize.rs

```diff
@@ -49,11 +49,14 @@ fn has_broken_link(text: &str) -> bool {
     new_cmark_parser(text, Some(&mut callback)).any(|event| {
         matches!(
             event,
-            Event::Start(Tag::Link(
-                LinkType::ReferenceUnknown | LinkType::CollapsedUnknown | LinkType::ShortcutUnknown,
-                _,
-                _
-            ))
+            Event::Start(Tag::Link {
+                link_type: LinkType::ReferenceUnknown
+                    | LinkType::CollapsedUnknown
+                    | LinkType::ShortcutUnknown,
+                dest_url: _,
+                title: _,
+                id: _
```

Comment:
I think this could be replaced with `..` like you do below. It's of course not an important difference :smile:

Reply:
Agreed, done.